lib/utils.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+const util = require('util');
+
+/**
+ * Forward-compatible unsafe allocation of buffer.
+ * @type {Function}
+ */
+const allocBufferUnsafe = Buffer.allocUnsafe;
```

Comment:
We can just use `Buffer.allocUnsafe` everywhere the `allocBufferUnsafe` is used an completely get rid of the utils file